src/GenerateAnalyzerRulesets/Program.cs

```diff
@@ -12,6 +12,8 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.VisualBasic.Syntax;
+using static System.Linq.ImmutableArrayExtensions;
+using static System.Collections.Immutable.ImmutableArrayExtensions;
```

Comment:
Why `using static` here and elsewhere?

Reply:
In fact I'm pretty sure that line 15 can be removed and the same extension method will still be resolved.

Reply:
This makes it double sure that that class' members are imported into the global scope.

It also signals developers that they should make sure these are the extension methods to use on `ImmutableArray`.

Reply:
> makes it double sure that that class' members are imported into the global scope

Can you give an example where this helps? I can't see how it'd influence overload resolution when the namespace containing the extension method is already imported.

Reply:
In this particular project? Probably not!

But if the utility types were in their own assembly, having `System.Collections.Immutable` would not force the existence of `System.Collections.Immutable.ImmutableArrayExtensions`.

Reply:
The assembly/project/package reference forces that, not the using declaration. They're redundant in this PR and I'd vote to remove them. Is there precedent for this pattern in the repo already?

Reply:
I agree with @drewnoakes

Reply:
The using declaration throws an error if the assembly/project/package is not referenced.

Reply:
Let us not over-engineer and keep it simple here - the shared project with these extensions/utilities will be referenced by each of our shipping analyzer packages, and we can assume these utilities are always available.